### PR TITLE
TASK-51342 : Edit task project after renaming a space leads to the loss of the project.

### PR DESCRIPTION
**Pull request by IlhemEssaadi**
ISSUES : Edit task project after renaming a space leads to the loss of the project.
FIX : the problem that the manager and participator variables did not take the right values ​​and which was solved by correcting the condition in order to take the right values.

### CODE REVIEW

File: task-management/src/main/webapp/vue-app/tasks-management/components/Project/ProjectAssigneeManager.vue [hunk 1]
```diff
@@ -149,7 +149,7 @@ export default {
       }
     },
     reset() {
-      if (this.manager && !this.manager.length>0) { // In case of edit existing event
+      if (this.manager && this.manager.length>0) { // In case of edit existing event
```
Comment (ahamdi) [inline task-management/src/main/webapp/vue-app/tasks-management/components/Project/ProjectAssigneeManager.vue]:
```suggestion
      if (this.manager && this.manager.length>0) {
```

**Review by ahamdi — CHANGES_REQUESTED** [review]

**Comment by rdenarie** [comment]
@IlhemEssaadi Ali request for a modification
Can you update please ?
Thanks

**Comment by IlhemEssaadi** [comment]
> @IlhemEssaadi Ali request for a modification Can you update please ? Thanks

already done!

**Comment by rdenarie** [comment]
> > @IlhemEssaadi Ali request for a modification Can you update please ? Thanks
> 
> already done!

Sorry, I didn't see it, my bad !

**Review by rdenarie — APPROVED** [review]

File: task-management/src/main/webapp/vue-app/tasks-management/components/Project/ProjectAssigneeManager.vue (hunk 1 above)
Comment (rdenarie) [reply task-management/src/main/webapp/vue-app/tasks-management/components/Project/ProjectAssigneeManager.vue]: @ahamdi can you validate the PR if ok ?
Thanks

**Review by ahamdi — APPROVED** [review]

**Pull request merged**